rosetta/identifier/operation.go

```diff
@@ -18,5 +18,5 @@ package identifier
 // needed because of the absence Flow does not support sharding.
 type Operation struct {
 	Index        uint `json:"index"`
-	NetworkIndex uint `json:"network_index"`
+	NetworkIndex uint `json:"network_index,omitempty"`
```

Comment:
This I spotted just now and I don't think it's okay - since it's probably valid to have the `network_index` as 0, but that shouldn't be omitted. Should probably be `*uint` like we did for the heights.

Reply:
Is there a case where we don't know the network index? Because ideally, we should always have it.

Reply:
This is also for the Construction API mainly - the Data API will have this.
But in the Construction API this does not make much sense, as we can populate it, but risk generating an output that is different from the one we receive.

Or we also enforce it on input (e.g. send is 0, receive is 1) and set it. But this is an optional field, feels weird to redeclare an optional field as mandatory.